.php_cs.dist

```diff
@@ -32,6 +32,16 @@ return PhpCsFixer\Config::create()
         'psr4' => false,
         'self_accessor' => false,
         'yoda_style' => null,
+        'align_multiline_comment' => [
+            'comment_type' => 'phpdocs_only',
+        ],
+        'array_indentation' => true, 
+        'combine_consecutive_issets' => true,
+        'combine_consecutive_unsets' => true,
+        'hash_to_slash_comment' => true,
```

Comment:
This has been deprecated, we should use `single_line_comment_style`

Reply:
Solved.